prometheus-metrics/src/main/scala/org/http4s/metrics/prometheus/Prometheus.scala

```diff
@@ -193,9 +279,11 @@ object Prometheus {
           F.delay {
             metrics.abnormalTerminations
               .labels(
-                label(classifier),
-                AbnormalTermination.report(AbnormalTermination.Canceled),
-                label(Option.empty),
+                List(
+                  label(classifier),
+                  AbnormalTermination.report(AbnormalTermination.Canceled),
+                  label(Option.empty),
+                ) ++ customLabels: _*
```

Comment:
These lists are small enough it probably doesn't matter, but we could optimize these a bit with a list buffer.

Reply:
Good catch, changed to use the more efficient `prepended` function.